index.html

```diff
@@ -152,14 +167,15 @@ <h5 class="topcenter">Profesional Art Printing Data</h5>
       </section>
     </section>
     <section class="about">
-      <h1>About me</h1>
-      <p>
-        Hello I’m a software developer! I can help you build a product , feature
-        or website Look through some of my work and experience! If you like what
-        you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-      <button>Get My Resume</button>
-      <img src="image/background-image.png" alt="hero image" />
+        <h1>About me</h1>
+        <p>
+          Hello I’m a software developer! I can help you build a product , feature
+          or website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
```

Comment:
- [ ] [OPTIONAL] kindly add the width of this section to make it more readable as it is a section that describes who you are.